site/_scss/blocks/_checkbox-group.scss

```diff
@@ -0,0 +1,44 @@
+checkbox-group {
+  label {
+    align-items: center;
+    color: get-color('grey-700');
```

Comment:
I won't block on this, since we do not support dark theme as of now. However, we should start thinking of adding universal color definitions (like you use --primary below) to support theming instead of definitive colors when we add new css.

Reply:
Cheers @devnook . Noted for future PRs